client/app/components/SelectWithVirtualScroll.tsx

```diff
@@ -32,7 +32,18 @@ function SelectWithVirtualScroll({ options, ...props }: VirtualScrollSelectProps
     return false;
   }, [options]);
 
-  return <AntdSelect<string> dropdownMatchSelectWidth={dropdownMatchSelectWidth} options={options} {...props} />;
+  if (options) {
+    optionFilterProp = "label";
+  }
```

Comment:
Since this component already use the `options` syntax to work, how about just set this prop below?

```jsx
<AntdSelect<string>
  dropdownMatchSelectWidth={dropdownMatchSelectWidth}
  options={options}
  optionFilterProp="label" {/* or after {...props} if you really want to force it to be "label" */}
  {...props}
/>
```